### PR TITLE
Remove different behaviour for listed SEFT surveys

### PR DESCRIPTION
**Pull request by AdamHawtin**
# Motivation and Context
We cannot currently reliably tell which collection exercises are SEFT versus EQ, the solution we had with a hardcoded list is not sustainable and needs to be removed. This PR removes all different behaviours for SEFT and removes the hardcoded list of ID's. The dashboard now treats all collection exercises the same.

# What has changed
- Removed list of SEFT survey ID's
- Removed all different behaviour for SEFT collection exercises

# How to test?
Run the dashboard, open a SEFT collection exercise. It should now show the same figures as the EQ dashboards.

### CODE REVIEW

File: app/static/assets/js/reporting.js [hunk 1]
```diff
@@ -79,16 +42,13 @@ function getReportEQ(response) {
...
 
     const timeUpdated = moment.unix(response.metadata.timeUpdated).calendar(); // eslint-disable-line
     let report = {};
 
-    if (collectionInstrumentType.toLowerCase() === "eq") {
-        report = getReportEQ(response);
-    } else {
-        report = getReportSEFT(response);
-    }
+    report = getReport(response);
```
Comment (MebinAbraham) [inline app/static/assets/js/reporting.js]: Can now be:
``` js
let report = getReport(response);
```

**Review by MebinAbraham — COMMENTED** [review]
Apart from the 1 minor comment, looks good.

File: tests/app/api/test_reporting.py [hunk 2]
```diff
@@ -22,34 +22,12 @@ def test_reporting_details_eq_success(self):
...
-
-            report = reporting_details('seft',
-                                       '57586798-74e3-49fd-93da-a782ec5f5129',
-                                       '14fb3e68-4dca-46db-bf49-04b84e07e999')
-            decoded_report = json.loads(report)
-
-        self.assertEqual(decoded_report, seft_reporting_response)
-
     def test_reporting_details_malformed_collex_id(self):
         response = self.test_client.get('/dashboard/reporting/eq'
```
Comment (ryangrundy7) [inline tests/app/api/test_reporting.py]: Collection instrument type is still on this test. It's not hitting anything in the repo

**Review by ryangrundy7 — CHANGES_REQUESTED** [review]
Collection instrument type is still on some of the tests which is causing the test coverage to get worse.

**Review by ryangrundy7 — APPROVED** [review]

File: scripts/mock_services.py [hunk 3]
```diff
@@ -18,7 +18,7 @@ def get_report(survey_id, collection_exercise_id):
 
     sample_size = rand_gen.randint(100, 1000)
     accounts_enrolled = rand_gen.randint(0, sample_size)
-    accounts_pending = rand_gen.randint(0, sample_size - accounts_enrolled)
+    accounts_pending = rand_gen.randint(0, (sample_size - accounts_enrolled)//10)
```
Comment (MebinAbraham) [inline scripts/mock_services.py]: Can you format to 
``` python
(sample_size - accounts_enrolled) // 10)
```
Standard style.

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**